cats_suite_helpers/cats_suite_helpers.go

```diff
@@ -48,6 +48,9 @@ func IsolationSegmentsDescribe(description string, callback func()) bool {
 			if Config.GetIsolationSegmentName() == "" {
 				Skip(`Skipping this test because Config.IsolationSegmentName is not set.`)
 			}
+			if Config.GetIsolationSegmentDomain() == "" {
```

Comment:
We think this would be better handled as a configuration error rather than skipping the test.  Could you add this functionality to the `Config` validation instead?  If the user is trying to `include_isolation_segments` then I think they would want to know early that they'd misconfigured the `integration_config.json`.